animal_years_months_converter/main.js

```diff
@@ -0,0 +1,274 @@
+import {animals} from "./js/animalData"
```

Comment:
The imported file is missing the file type
```suggestion
import {animals} from "./js/animalData.js"
```